src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Fixed a bug where `TracerProviderBuilderBase` was not invoking the
+  `instrumentationFactory` delegate passed to the `protected`
+  `AddInstrumentation` method.
+  ([#4873](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4873))
```

Comment:
```suggestion
  ([#4873](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4873))
```

Reply:
It seems there are other entries in the CHANGELOG which need to be fixed.

Reply:
Updated the one I added and all the ones at the ~top using `/issues/` path instead of `/pull/`.